getting-started.md

```diff
@@ -83,6 +83,54 @@ If everything works, you should see your starter app on your device or simulator
   </div>
 </div>
 
+## _Hot Reloading_ and _Full Application Restart_ from within IntelliJ
```

Comment:
Generally I think much of this content belongs in the intellij-ide.md topic, not here in getting started. We want to keep getting started really short and focused on just the essentials.

Concretely I suggest keeping here just lines 86 through 97, and then moving the rest.